src/utilities/color-transformers.ts

```diff
@@ -213,32 +213,25 @@ export function hexToRgb(color: string) {
   return {red, green, blue};
 }
 
-export enum ColorType {
-  Hex = 'hex',
-  Rgb = 'rgb',
-  Rgba = 'rgba',
-  Hsl = 'hsl',
-  Hsla = 'hsla',
-  Default = 'default',
-}
+type ColorType = 'hex' | 'rgb' | 'rgba' | 'hsl' | 'hsla' | 'default';
```

Comment:
Not feeling strongly about this change, but in VS Code, I think enums are a tad easier to trace back ("show all implementations/references").

Reply:
Over the past week I've been rallying against enums as I've learned that [how they get expressed at runtime is pretty clunky when compared to using union types](https://www.typescriptlang.org/play/#code/KYOwrgtgBAKsDOAXAouCBlRAnAliA5lAN4CwAUFFAGoCGANmMAIxQC8UA5AAwcA05lWg2AAmNpyZ9yAX3LkAZmBABjRDgD2IKIgSJMuAgAocOiAC5Yu1JH158ASmICo8AO4nlACyjHTj0hSUUMo08MCWSNYY2HYAdEKMTBZYwIhgWFocNBwA3M6UIWERKGi2BPH0jCLJqemZAEa5zrJkLeQ6SGX4hnCRpTHlCcz2eWTkAPTj1OhyZIgAngAO4b2IAKogGiAwS+Hs3BwAPhIcs4oqapraul0ivsDmxRtbO8v+zm4e3vcQ74EFoXCBxqaQynGyoyCwUBJxBdU4jUhUBabTmNwG+DuBxG5CAA), and in addition to that they're not treeshakable as defining an enum doesn't emit code that is unambiguously pure - because [enums may be extended later](https://www.typescriptlang.org/play/?ssl=10&ssc=40&pln=1&pc=1#code/KYOwrgtgBAkgLsCBnKBvAsAKClAagQwBsxgBGKAXigHIAGagGixwOOACZKbTGsBfLFlCRYCZGijM8REgGYu1dtX6DMAYwD2IJBsLAAdIQ0BzABTxESfaxKkGoy9ZnBZASiA)(I'm filing this under "thanks, I hate it").

In some magical ideal world it's be great if enums could compile down to simple objects, or `const enum` was usable in a single file in isolatedModules mode, but that's not how things work currently unfortunately.